src/ducks/SANCharts/data.js

```diff
@@ -398,8 +400,34 @@ export const Metrics = {
     node: 'line',
     Component: Line,
     label: 'Twitter',
-    // dataKey: 'followersCount',
-    description: `Shows the number of followers on the project's official Twitter account over time`
+    description: `Shows the number of followers on the project's official Twitter account over time`,
+    subMetrics: SOCIAL_TWITTER_INTERVALS.map(interval => {
```

Comment:
Submetrics should be placed in the different file, since: 1) file length; 2) it will be soon decoupled in a similar manner; 3) performance reasons. I suggest, for now, to put it in a `../SANCharts/submetrics.js`. 

Metric's submetrics array will be accessed via metric's key.

```js
export const Submetric = {
  'twitterKey': [
    { 
      ...twitterMetric,
      key: 'twitter_followers_7d',
      queryKey: 'twitter_followers',
      label: 'Twitter Changes (7d)',
      reqMeta: {
        interval: '7d',
        transform: {...}
      }
    }
  ]
}
```

Reply:
good mind